server/base/src/main/java/org/apache/accumulo/server/metadata/iterators/SetEqualityIterator.java

```diff
@@ -45,16 +47,22 @@
 
 /**
  * This iterator exists to enable checking for set equality in a conditional mutation. It allows
- * comparing a set in a client process to a set encoded in column qualifiers within a tablet.
+ * comparing a set in a client process to a set encoded in column qualifiers within a tablet. If the
+ * "concat.value" options is supplied and is true, then the bytes from the Value will be added with
+ * a null byte separator.
```

Comment:
Could do something like the following. This better allows for addition of more opts in the future and separates them from the description.
```suggestion
 * comparing a set in a client process to a set encoded in column qualifiers within a tablet.
 *
 * <h3>Options</h3>
 * <ul>
 * <li><b>concat.value:</b> If this option is supplied and is true, then the bytes from the Value
 * will be concatenated with a null byte separator.</li>
 * </ul>
```